### ISSUE TITLE
Feature request: display last successful update date

### ISSUE THREAD

**Issue by vi** [syncthing/syncthing-android#523]
Supposing Syncthing scanned a folder at datetime X, then successfully and fully synced the folder to at least one device at datetime Y, I want X to be visible for the folder in GUI.

Now it just shows "up to date", but I'm not sure up to _which_ date is it.

**Comment by AudriusButkevicius** [comment]
Last received file has a date when you over on it.

**Comment by vi** [comment]
I mean the date of sent data, not received one. To answer the question "Is this device actually backed up now?".

Rescan interval can be big, and it's not clear whether everything is sent or hasn't just yet detected the changes.

**Comment by AudriusButkevicius** [comment]
Well you have completion percentages which says whether it's up to date or not.
It's still not clear to me what you are after.

**Comment by vi** [comment]
If I see 96%, then 100% then I'm more or less sure that it's recently updated.

But if I see just 100% then it's not certain whether it is just updated or a week ago (and hasn't rescanned since then).

**Comment by AudriusButkevicius** [comment]
As I said, you can hover on `Last file received`, and it will tell you when was the last time it has received a file.

**Comment by vi** [comment]
I suppose this is about receiving data (i.e. backuing up peer device, not this device).

I want to know about sending (i.e. is data on this device has successfully been transferred to remote backup device).

**Comment by AudriusButkevicius** [comment]
I guess what you are asking for is, when did peers last time exchanged their state.

**Comment by vi** [comment]
Probably yes.

That may be accompanied by some warning if peer is offline or fails to synchronize for a long enough time (i.e. auto-backup is not happening).

**Comment by AudriusButkevicius** [comment]
No, that's out of scope. Syncthing is not for backups.

**Comment by licaon-kter** [comment]
@AudriusButkevicius What do you use it for?

**Comment by AudriusButkevicius** [comment]
I don't.

**Comment by Nutomic** [comment]
I think this issue should really go to [syncthing](https://github.com/syncthing/syncthing/issues) directly, because it's not something we can do in the app.

**Comment by AudriusButkevicius** [comment]
Welp, remote index updated event is enough.

**Comment by vi** [comment]
Does that "remote index updated" event mean that all files are settled at the remote side?

If upload process is interrupted in the middle (by walking away from Wi-Fi, for example), it should show old update date.

**Comment by Catfriend1** [comment]
@vi @AudriusButkevicius Isn't it the LocalIndexUpdated event we would need in case we want to show when a folder was last scanned?

**Comment by AudriusButkevicius** [comment]
No, that just implies something was scanned, not necesserily sent to someone, which I think is what this PR is asking for. RemoteIndexUpdated could mean remote side scanned and discovered something or downloaded something (which I guess is close enough, as usually means you are online and talking to someone who might have pulled a file from you at the time).

**Comment by Catfriend1** [comment]
Hmm ... ok I'd thought we would stick to the syncthing Web UI and show folder last scanned time for every folder in the app UI. One folder can have multiple associated devices, so which RemoteIndexUpdated datetimestamp would fit here, thats unclear to me.

**Comment by vi** [comment]
It should allow estimating how much of data would be lost if the device is suddenly destroyed/lost now.

**Comment by Catfriend1** [comment]
So as @AudriusButkevicius said. the information and percentages are already there on the web UI. Should we close this or try the "RemoteIndexUpdated" visualization approach?
